src/ashell/shell-state.c

```diff
@@ -644,12 +618,14 @@ static const struct ashell_cmd commands[] =
 
 int32_t ashell_help(char *buf)
 {
-    comms_println("'A Shell' bash\r\n");
-    comms_println("Commands list:");
+    comms_print("'A Shell' bash\r\n\r\n");
+    comms_print("Commands list:\r\n");
     for (uint32_t t = 0; t < ASHELL_COMMANDS_COUNT; t++) {
-        comms_printf("%8s %s\r\n", commands[t].cmd_name, commands[t].syntax);
+        comms_print(commands[t].cmd_name);
```

Comment:
Again using comms_print and comms_write_buf instaed of comms_printf to avoid extra chars.